roles/network/tasks/enable_services.yml

```diff
@@ -50,9 +51,15 @@
 - name: Install /etc/dnsmasq.conf from template, if dnsmasq_enabled
   template:
     src: network/dnsmasq.conf.j2
-    dest: /etc/dnsmasq.conf
+    dest: /etc/dnsmasq.d/iiab.conf
   when: dnsmasq_enabled and dnsmasq_install
```

Comment:
```suggestion
  when: dnsmasq_enabled and dnsmasq_install and (iiab_network_mode != "Appliance")
```

Reply:
won't that break the parsing when not aligned under the 'name'?